src/pages/admin/login.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import {
+  FormLabel,
+  FormControl,
+  Input,
+  Button,
+  Heading,
+  FormErrorMessage,
+} from '@chakra-ui/react';
+
+import styles from '@/styles/Admin.module.css';
+
+export default function Admin() {
+  const {
+    handleSubmit,
+    register,
+    formState: { errors, isSubmitting },
+  } = useForm();
+
+  function onSubmit() {
+    return new Promise<void>((resolve) => {
+      setTimeout(() => {
+        resolve();
+      }, 3000);
+    });
+  }
+
+  return (
+    <div className={styles.container}>
+      <Heading mb="2vh">Login</Heading>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <FormControl isInvalid={errors.username} mt={2} mb={2}>
+          <FormLabel htmlFor="username">Username</FormLabel>
+          <Input
+            id="username"
+            placeholder="Username"
+            {...register('username', {
+              required: 'This is required',
+            })}
+            borderColor="black"
+            borderRadius="-moz-initial"
```

Comment:
I don't think this does anything, also not a good practice to use only use one browser-specific CSS value

Reply:
I think this was added to change the border radius from rounded to squared out

Reply:
yeah that's fine, the design is a wireframe so it doesn't have to be followed very exactly.